bin/validate-payload-examples.ts

```diff
@@ -3,7 +3,7 @@
 import { DefinedError, ErrorObject } from "ajv";
 import path from "path";
 import { inspect } from "util";
-import { ajv, validate } from "../payload-schemas";
+import { ajv, validate } from "../payload-schemas/index";
```

Comment:
```suggestion
import { ajv, validate } from "../payload-schemas";
```

nit: node imports `index` if it exists when you tell it to import a folder, meaning you can leave this file as is.

Reply:
Yes, it usually does, but in this case, since there's a `package.json` in the directory with a `main` property, it'll use the `main` property which in this case is `schema.json`.

Which is why I used an explicit import instead of implicit